.github/workflows/lint.yaml

```
@@ -37,3 +37,19 @@ jobs:
         run: make lint
+
+  lint-operator:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
```

actionlint (1.7.8)

the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)